lib/matplotlib/tests/test_backend_pgf.py

```diff
@@ -83,8 +82,8 @@ def create_figure():
 
 # test compiling a figure to pdf with xelatex
 @needs_xelatex
-@cleanup(style='classic')
-@switch_backend('pgf')
+@pytest.mark.style('default')
```

Comment:
For those reading along at home, the outer decorator that was here was being ignored and the last time we generated the images for these test we accidentally switched to the default style.

This was 'fixed' as part of the pytest migration so we need to switch the mark to `'default'` style.